app/src/main/java/com/jemiola/moodtimeline/views/timeline/TimelneaiModelImpl.kt

```diff
@@ -0,0 +1,14 @@
+package com.jemiola.moodtimeline.views.timeline
+
+class TimelneaiModelImpl : TimelienModel {
+
+    var animDuration = 100
+
+    fun getAnimDurationn(): Int {
```

Comment:
Jakbym sobie chciała gettery i settery robic to bym pisała w Javie. Polecam zaznajomić się z lekturą:
https://www.soldevelo.com/blog/why-you-should-use-kotlin-instead-of-java-in-your-next-project/